debian/packetfence.preinst

```diff
@@ -64,6 +64,8 @@ case "$1" in
         echo "RateLimitInterval=0" >> /etc/systemd/journald.conf
         echo "RateLimitBurst=0" >> /etc/systemd/journald.conf
         echo "ForwardToWall=no" >> /etc/systemd/journald.conf
+        echo "SystemMaxUse=1G" >> /etc/systemd/journald.conf
```

Comment:
Can't we use the same logic as RHEL/CentOS and use the provided file rather than writing a new one ?
That would avoid having to "manage" two separate configurations